src/memspaces/memspace_host_all.c

```diff
@@ -40,7 +40,10 @@ static umf_result_t umfMemspaceHostAllCreate(umf_memspace_handle_t *hMemspace) {
     // object that can be present on multiple levels.
```

Comment:
should you also update the comment here...?

Reply:
please propose how it should be changed

Reply:
We changed from checking `!= -1` to `<0`, so I ghouth we could update the part about `-1`... but I think I got ahead of myself, this func can only return `-1` and `0`, so no need to change the comment, after all.